Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12-alpine
+
+RUN apk add --no-cache mariadb-dev make g++ linux-headers nodejs tzdata
+
+WORKDIR /website
+
+COPY package.json .
+COPY package-lock.json .
+
+RUN npm install
+RUN npm install nuxt --global
```

Comment:
Do we want to specify the nuxt version here?

Reply:
Ah good point.